README.md

```diff
@@ -236,6 +236,25 @@ case it's a new package, be sure to add it to the correct koji tags as well.  [H
 
 * This uses local filesystem, so please be sure your using a clean checkout with no local changes
 
+## Releasing Katello client packages for SUSE
+
+For SLES client builds, they must be manually done for now by submitting the jobs to [OBS](https://build.opensuse.org/).  You can use the OSC client tools.
+
+Create an ~/.oscrc with your credentials, and checkout an OBS repo. For example:
+
+```
+osc checkout home:stbenjam katello-repos
```

Comment:
Would it be good to use `home:$USER` as an example instead?

Reply:
Yea that'd work. I was hoping we'd create a `katello` account before this got merged TBH.  I'll ask today

Reply:
Overall this looks fine, but yeah lets get a common 'project' that can have multiple contributors